test_project/test_project/settings.py

```diff
@@ -142,6 +145,20 @@
 
 GRAPPELLI_ADMIN_TITLE = "Dancerfly"
 
+STATICFILES_FINDERS = (
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+    # other finders..
+    'compressor.finders.CompressorFinder',
+)
+
+_BOOTSTRAP_SASS_SUBPATH = "/gems/bootstrap-sass-3.3.4.1/assets/stylesheets/"
+STATICFILES_DIRS = [x + _BOOTSTRAP_SASS_SUBPATH for x in os.environ['GEM_PATH'].split(":") if os.path.isdir(x + _BOOTSTRAP_SASS_SUBPATH)]
+
+COMPRESS_PRECOMPILERS = (
+    ('text/sass', 'django_libsass.SassCompiler'),
```

Comment:
I believe either mimetype is acceptable. Sass isn't responsible for compression. I agree that it'd be nice not to compress in development (I think Stephen turned it on). You just need to switch COMPRESS_ENABLED to False. Or maybe just set it to not DEBUG or something.

On Tue, Apr 14, 2015 at 12:01 PM, Naomi I. Morduch Toubman
<notifications@github.com> wrote:

>> @@ -142,6 +145,20 @@
>>  
>>  GRAPPELLI_ADMIN_TITLE = "Dancerfly"
>>  
>> +STATICFILES_FINDERS = (
>> +    'django.contrib.staticfiles.finders.FileSystemFinder',
>> +    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
>> +    # other finders..
>> +    'compressor.finders.CompressorFinder',
>> +)
>> +
>> +_BOOTSTRAP_SASS_SUBPATH = "/gems/bootstrap-sass-3.3.4.1/assets/stylesheets/"
>> +STATICFILES_DIRS = [x + _BOOTSTRAP_SASS_SUBPATH for x in os.environ['GEM_PATH'].split(":") if os.path.isdir(x + _BOOTSTRAP_SASS_SUBPATH)]
>> +
>> +COMPRESS_PRECOMPILERS = (
>> +    ('text/sass', 'django_libsass.SassCompiler'),
> Why do we need this? The example in the docs has `('text/x-sass', 'sass {infile} {outfile}')`. (I'm wondering about sass compiler settings, because it would be nice to have it compile uncompressed in development.)
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/littleweaver/django-brambling/pull/440/files#r28341057